### PR TITLE
BaselOne conference report

### PR DESCRIPTION
**Pull request by madmas**

### CODE REVIEW

**Review by madmas — COMMENTED** [review]
extend the article a little further

**Review by madmas — COMMENTED** [review]
extend the article further

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Some figures about BaselOne22 
+BaselOne is a yearly event organized by the eponymous association existing of [Karakun AG](https://karakun.com), [Bluesky IT-Solutions AG](https://bluesky-it.ch) and [Baloise Insurance](https://baloise.com).
+
+In total, BaselOne 22 offered 26 talks by 29 speakers and 4 workshops to 160 paying attendees. All in all, almost 400 hours of organization time were spent to make it happen.
+
+
+
+## Day 1 - Conference Day
+
+As said, there were 26 talks, and it was challenging to choose which one to attend. The lineup was packed with well-known speakers like Ixchel Ruiz, Falk Sippach, Sandra Parsick, Otavio Santana, Katrin Rabow, Gerd Aschemann, Chris Engelbert and many more. The talks were nicely distributed across variety of topics and domains of IT. It is worth to mention that most of the talks had an introductory flavour to it to attract more beginners, but also to light a spark among more experienced folks. From more advanced talks to one of our colleague [George](https://twitter.com/grigala_) clearly stand out a talk about Service Mesh by [Abdellfetah Sghiouar](https://twitter.com/boredabdel):
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:25]: I suggest linebreaks and think the last sentence had a typo:
```suggestion
As said, there were 26 talks, and it was challenging to choose which one to attend. 
The lineup was packed with well-known speakers like Ixchel Ruiz, Falk Sippach, Sandra Parsick, Otavio Santana, Katrin Rabow, Gerd Aschemann, Chris Engelbert and many more.
The talks were nicely distributed across variety of topics and domains of IT. 
It is worth to mention that most of the talks had an introductory flavour to it to attract more beginners, but also to light a spark among more experienced folks. 
From the more advanced talks to one of our colleague [George](https://twitter.com/grigala_) clearly stands out a talk about Service Mesh by [Abdellfetah Sghiouar](https://twitter.com/boredabdel):
```

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+<blockquote class="twitter-tweet" style="justify-content: center"><p lang="en" dir="ltr">Very interesting talk by <a href="https://twitter.com/boredabdel?ref_src=twsrc%5Etfw">@boredabdel</a> <a href="https://twitter.com/Baselone?ref_src=twsrc%5Etfw">@Baselone</a> about Service Mesh with <a href="https://twitter.com/IstioMesh?ref_src=twsrc%5Etfw">@IstioMesh</a> <a href="https://twitter.com/kubernetesio?ref_src=twsrc%5Etfw">@kubernetesio</a> <a href="https://twitter.com/hashtag/BaselOne22?src=hash&amp;ref_src=twsrc%5Etfw">#BaselOne22</a> <a href="https://twitter.com/hashtag/BaselOne?src=hash&amp;ref_src=twsrc%5Etfw">#BaselOne</a> <a href="https://t.co/4Q2zZ2Vqaa">pic.twitter.com/4Q2zZ2Vqaa</a></p>&mdash; George Grigalashvili (@grigala_) <a href="https://twitter.com/grigala_/status/1582664710763053056?ref_src=twsrc%5Etfw">October 19, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:27]: I personally don't like to load the script from twitter which als might be in conflict with data protection, what do the others think?

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+The talk explored how service mesh by the example of [Istio](https://istio.io/latest/) helps with many common challenges of a distributed architecture.
+
+![Alexander Schwarz, RedHat](/assets/posts/2022-10-25-basel-one/Basel-One-29.jpg)
+
+And of course, there were also talks by Karakun: "Runtime metrics 101" (Markus Schlichting / Stephan Classen) and "Ich bin 3-CPO - wie kann ich Dir helfen?" (Iryna Dohndorf / Christopher Keibel).
+
+### Runtime Metrics 101
+
+As modern applications are no longer monoliths on a server but consist of distributed microservices in the cloud, monitoring becomes more and more important. Logging alone won't be enough if you want to be informed about free disk space, a critical number of user sessions or increasing memory consumption. This is where runtime metrics can help.
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:37]:
```suggestion
As modern applications are no longer monoliths on a server but consist of distributed microservices in the cloud, monitoring becomes more and more important.
 Logging alone won't be enough if you want to be informed about free disk space, a critical number of user sessions or increasing memory consumption. 
 This is where runtime metrics can help.
```

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+During their session, Markus and Stephan showed the audience how to implement, visualize and analyze metrics for Java applications. They did not only focus on products and ready-made solutions like [Grafana](https://grafana.com) or [Prometheus](https://prometheus.io). Furthermore, they also presented different APIs like Java Flight Recorder and [Micrometer](https://micrometer.io) that can be used to extend a Java application with metrics and continuous monitoring.
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:39]:
```suggestion
During their session, Markus and Stephan showed the audience how to implement, visualize and analyze metrics for Java applications. 
They did not only focus on products and ready-made solutions like [Grafana](https://grafana.com) or [Prometheus](https://prometheus.io). 
Furthermore, they also presented different APIs like Java Flight Recorder and [Micrometer](https://micrometer.io) that can be used to extend a Java application with metrics and continuous monitoring.
```

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+![Stephan explaining metrics](/assets/posts/2022-10-25-basel-one/IMG_1530.jpeg)
+
+The slides are available [here](https://dev.karakun.com/assets/talks/Slidedeck_RuntimeMetriken101_BaselOne.pdf). The talk is also available at [Karakun YouTube Channel](https://karakun.com/youtube) or via this [direct link](https://youtu.be/XBtFGwqy7yE).
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:43]:
```suggestion
The slides are available [here](https://dev.karakun.com/assets/talks/Slidedeck_RuntimeMetriken101_BaselOne.pdf). 
The talk is also available at [Karakun YouTube Channel](https://karakun.com/youtube) or via this [direct link](https://youtu.be/XBtFGwqy7yE). 
```

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+During their session, Markus and Stephan showed the audience how to implement, visualize and analyze metrics for Java applications. They did not only focus on products and ready-made solutions like [Grafana](https://grafana.com) or [Prometheus](https://prometheus.io). Furthermore, they also presented different APIs like Java Flight Recorder and [Micrometer](https://micrometer.io) that can be used to extend a Java application with metrics and continuous monitoring.
+
+![Stephan explaining metrics](/assets/posts/2022-10-25-basel-one/IMG_1530.jpeg)
+
+The slides are available [here](https://dev.karakun.com/assets/talks/Slidedeck_RuntimeMetriken101_BaselOne.pdf). The talk is also available at [Karakun YouTube Channel](https://karakun.com/youtube) or via this [direct link](https://youtu.be/XBtFGwqy7yE). 
+
+### "Ich bin C-3PO, wie kann ich Dir helfen?" - Wie Maschinen natürliche Sprache verarbeiten 
+
+This talk by Iryna and Christopher was all about *Natural Language Processing (NLP)*. We all use NLP in our daily lifes e.g. when using Siri, Alexa and search engines. But there are far more things to come in the future.
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:47]:
```suggestion
This talk by Iryna and Christopher was all about *Natural Language Processing (NLP)*.
We all use NLP in our daily lifes e.g. when using Siri, Alexa and search engines. 
But there are far more things to come in the future. 
```

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+![Stephan explaining metrics](/assets/posts/2022-10-25-basel-one/IMG_1530.jpeg)
+
+The slides are available [here](https://dev.karakun.com/assets/talks/Slidedeck_RuntimeMetriken101_BaselOne.pdf). The talk is also available at [Karakun YouTube Channel](https://karakun.com/youtube) or via this [direct link](https://youtu.be/XBtFGwqy7yE). 
+
+### "Ich bin C-3PO, wie kann ich Dir helfen?" - Wie Maschinen natürliche Sprache verarbeiten 
+
+This talk by Iryna and Christopher was all about *Natural Language Processing (NLP)*. We all use NLP in our daily lifes e.g. when using Siri, Alexa and search engines. But there are far more things to come in the future. 
+
+NLP uses linguistics and machine learning to get a general language comprehension which will be quite similar to human cognitive abilities. In this session, Iryna and Christopher presented the tools and techniques of NLP and discussed the roles of machine learning and artificial neural networks for optimization of NLP.
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:49]:
```suggestion
NLP uses linguistics and machine learning to get a general language comprehension which will be quite similar to human cognitive abilities. 
In this session, Iryna and Christopher presented the tools and techniques of NLP and discussed the roles of machine learning and artificial neural networks for optimization of NLP. 
```

File: _posts/2022-10-26-basel-one-conference-report.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+The slides are available [here](https://dev.karakun.com/assets/talks/Slidedeck_RuntimeMetriken101_BaselOne.pdf). The talk is also available at [Karakun YouTube Channel](https://karakun.com/youtube) or via this [direct link](https://youtu.be/XBtFGwqy7yE). 
+
+### "Ich bin C-3PO, wie kann ich Dir helfen?" - Wie Maschinen natürliche Sprache verarbeiten 
+
+This talk by Iryna and Christopher was all about *Natural Language Processing (NLP)*. We all use NLP in our daily lifes e.g. when using Siri, Alexa and search engines. But there are far more things to come in the future. 
+
+NLP uses linguistics and machine learning to get a general language comprehension which will be quite similar to human cognitive abilities. In this session, Iryna and Christopher presented the tools and techniques of NLP and discussed the roles of machine learning and artificial neural networks for optimization of NLP. 
+
+The slides are available at the [BaselOne website](https://baselone.ch/dam/baselone2022/assets/Slides2022/BaselOne2022_C-3PO.pdf). The talk is also available at [Karakun YouTube Channel](https://karakun.com/youtube) or via this [direct link](https://youtu.be/Z47sGjK5hXc).
```
Comment (madmas) [inline _posts/2022-10-26-basel-one-conference-report.md:51]:
```suggestion
The slides are available at the [BaselOne website](https://baselone.ch/dam/baselone2022/assets/Slides2022/BaselOne2022_C-3PO.pdf). 
The talk is also available at [Karakun YouTube Channel](https://karakun.com/youtube) or via this [direct link](https://youtu.be/Z47sGjK5hXc).
```

**Review by madmas — COMMENTED** [review]
@grigala @sclassen please have a look at my suggestions ;-) 
I think, for your addition Stephan there are only line breaks ;-)

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 2 above)
Comment (sclassen) [reply _posts/2022-10-26-basel-one-conference-report.md:27]: no twitter for me

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 1 above)
Comment (sclassen) [reply _posts/2022-10-26-basel-one-conference-report.md:25]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 3 above)
Comment (sclassen) [reply _posts/2022-10-26-basel-one-conference-report.md:37]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 4 above)
Comment (sclassen) [reply _posts/2022-10-26-basel-one-conference-report.md:39]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 5 above)
Comment (sclassen) [reply _posts/2022-10-26-basel-one-conference-report.md:43]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 7 above)
Comment (sclassen) [reply _posts/2022-10-26-basel-one-conference-report.md:49]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 8 above)
Comment (sclassen) [reply _posts/2022-10-26-basel-one-conference-report.md:51]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 2 above)
Comment (grigala) [reply _posts/2022-10-26-basel-one-conference-report.md:27]: Remove if so, I don't have any problem, just wanted to highlight things. I don't understand that data protection crap for the content that is publicly available.

Comment (madmas) [reply _posts/2022-10-26-basel-one-conference-report.md:27]: the data protection thing is that twitter spies on the user with that script. so if we load it from there embedd in out page we need to let the user know and decide if he is okay with that. does that help?

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 1 above)
Comment (kress04) [reply _posts/2022-10-26-basel-one-conference-report.md:25]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 2 above)
Comment (kress04) [reply _posts/2022-10-26-basel-one-conference-report.md:27]: Is it really true that Twitter can notice the IP of the visitor if the tweet is embedded in the page? I would doubt that. A visitor's IP is logged in Matomo (our web analytics) but why should it be forwarded to Twitter? 

Anyway, it is better to be cautious in this regard...

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 3 above)
Comment (kress04) [reply _posts/2022-10-26-basel-one-conference-report.md:37]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 4 above)
Comment (kress04) [reply _posts/2022-10-26-basel-one-conference-report.md:39]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 6 above)
Comment (kress04) [reply _posts/2022-10-26-basel-one-conference-report.md:47]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 7 above)
Comment (kress04) [reply _posts/2022-10-26-basel-one-conference-report.md:49]: lgtm

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 8 above)
Comment (kress04) [reply _posts/2022-10-26-basel-one-conference-report.md:51]: lgtm

**Comment by kress04** [comment]
I hope we're not finished yet with the article. We should add something like a conclusion, statement, whatever at the end.

File: _posts/2022-10-26-basel-one-conference-report.md (hunk 2 above)
Comment (grigala) [reply _posts/2022-10-26-basel-one-conference-report.md]: Removed and adopted sentence per line rule for this paragraph. 

Will keep my extremist views regarding "data protection" nonsense for myself. 😁

Comment (madmas) [reply _posts/2022-10-26-basel-one-conference-report.md]: @kress04 it is as there is a request from the users browser to twitter.com when it loads the script. And as it is a JavaScript, it potentially logs a lot more than just the IP.

@grigala I often feel the same on the one hand, but on the other hand, data protection protects in most of the cases us as individuals so it´s worth to accept it ;-))

**Review by madmas — COMMENTED** [review]
draft for a conclusion

**Pull request merged**